workers/unity/Packages/com.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -20,20 +20,33 @@ internal static class GenerateCode
         private static readonly string SchemaCompilerRelativePath =
             $"../build/CoreSdk/{Common.CoreSdkVersion}/schema_compiler/schema_compiler";
 
+        private static readonly string StartupCodegenMarkerFile =
+            Path.GetFullPath("Temp/ImprobableCodegen.marker");
```

Comment:
`/`, may be better to use `Path.Join`

Reply:
I think it's safe to use `"Temp"` here but may be even safer to do e.g. `Path.Join(Application.dataPath, "..", "Temp", "filename.ext")`

Reply:
if this is the right folder, we should use this:
https://docs.unity3d.com/ScriptReference/Application-temporaryCachePath.html

Reply:
This one could potentially clash with other projects:

`%USERPROFILE%/AppData/Local/Temp/{company name}/{project name}`

Where {project name} is likely `unity` and {company name} is likely `DefaultCompany`

Reply:
That shouldn't be the windows temp, but the project temp

Reply:
There's an easy way to test this 😉

Reply:
It is indeed - `%USERPROFILE%/AppData/Local/Temp/{company name}/{project name}`

Reply:
It also doesn't get cleaned when Unity closes or opens. Am going to revert back to the original

Reply:
re: the original: reminder about my comments about Path.Join